paclair/docker/docker_registry.py

```diff
@@ -106,20 +110,25 @@ def get_token(self, docker_image):
         :param docker_image: paclair.docker.DockerImage
         :returns str:
         """
-        # Define url
-        url = self.token_url.format(registry=self, image=docker_image)
-        self.logger.debug("REQUEST_TOKEN:URL:{url}".format(url=url))
-
-        # Get token
-        resp = requests.get(url, verify=self.verify, auth=self.auth)
-        if not resp.ok:
-            self.logger.error("REQUEST_TOKEN:HTTPCODEERROR:{}".format(resp.status_code))
-            raise RegistryAccessError("Error access to : {url} \nCode Error : {status_code}".format(
-                url=url, status_code=resp.status_code))
-
-        token = resp.json()['token']
-        self.logger.debug("TOKEN:{token}".format(token=token))
-        return token
+        if self.__token is None:
```

Comment:
Return immediately self.__token if it is not None. It keeps the code more clear and concise.
```
if self.__token is not None:
    return self.__token
```